jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/publisher/impl/AnnotationPublisher.java

```diff
@@ -486,9 +483,10 @@ private String buildIssueKey(Map<String, String> resource) {
      *
      * @param responseMapList the response map list
      */
-    private void processErrors(List<Map<String, Map>> responseMapList) {
-        logger.error("some errors occured while publishing the anotation, but no error handler found to handle it",
+    private void processErrors(List<Map<String, Map>> responseMapList, String response) {
+        logger.error("some errors occurred while publishing the annotation, but no error handler found to handle it",

```

Comment:
can we be more specific? not `some` but particular number with details?

Reply:
this logger is present from earlier. i just fixed spelling of words 'annotation' and 'occurred'. Please see line 488, 489. We are already trying to print some details.

Reply:
okay done.